### PR TITLE
Reezan

### PR DESCRIPTION
**Pull request by Shinigami-7**
commented each code

### CODE REVIEW

File: Webscraping/sel.py [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 
+options = webdriver.FirefoxOptions()  # -> This allow to access the firefox web browser
 
-options = webdriver.FirefoxOptions()
-options.add_argument("-headless")
+options.add_argument("-headless")   # -> This run browser in background
 
 
-driver = webdriver.Firefox(options=options)
+driver = webdriver.Firefox(options=options) # ->it provide  argument to driver whic run browser in backgroung
```
Comment (jyotsanh) [inline Webscraping/sel.py]: # Mr.Reezan
### the  spelling of 'which' is mistake can you look into it.
### at line 13 comment section pls i ask you to check again

File: Webscraping/sel.py [hunk 2]
```diff
@@ -55,18 +56,26 @@ def page_source_to_dataframe(page_html):
...
+driver.get("https://www.sharesansar.com/company/nabil")
 
-driver.get("https://www.sharesansar.com/company/hidcl")
-
+# -> 'price history' button x_path which driver uses to click
 price_history_xpath = "/html/body/div[2]/div/section[2]/div[3]/div/div/div/div[2]/div/div[1]/div[1]/ul/li[8]"
 button = driver.find_element(By.XPATH,price_history_xpath)
+# -> driver click button
 button.click()
+
```
Comment (jyotsanh) [inline Webscraping/sel.py]: ## from line 53 i don't see the comments being used why is  that .

**Review by Shinigami-7 — COMMENTED** [review]
thank you for pointing that out

**Review by jyotsanh — APPROVED** [review]
it looks better now.

**Pull request merged**